### DISCUSSION TITLE
Add a couple of bots to our TACO fork

### DISCUSSION THREAD

**Discussion by guilhermeleobas** [pydata/sparse#483] [Ideas]
I was thinking to add a few bots to perform some tasks in our taco fork:
- Sync our fork with taco main branch: https://github.com/wei/pull
- clang-format for llvm related files: https://github.com/marketplace/actions/clang-format-lint

What do you think, @hameerabbasi?

**Comment by hameerabbasi** [comment]
Sounds good to me. 😄
